### ISSUE TITLE
restore window to last known geometry after full screen

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#105]
when disabling full screen by hitting F11, the window should be returned to the same width, height, and possibly position as it was before enabling full screen
